bismuth.spec

```diff
@@ -12,6 +12,7 @@ BuildRequires:  cmake ninja-build extra-cmake-modules npm
 BuildRequires:  kf5-kconfigwidgets-devel qt5-qtbase-devel qt5-qtbase-private-devel
 BuildRequires:  qt5-qtdeclarative-devel qt5-qtquickcontrols2-devel qt5-qtsvg-devel 
 BuildRequires:  qt5-qtfeedback-devel kf5-kcmutils-devel kf5-ki18n-devel kf5-kdeclarative-devel
+BuildRequires:	kdecoration-devel kf5-kglobalaccel-devel
```

Comment:
You used a tab here but it should be a space, don't get me wrong I'm actually a tabs person but spec files seem to all use spaces.

You should also run `rpmlint` on the spec file since it will provide some best practices and warn about things like this (mixed style, which IMO makes sense stick with one even if you prefer the other).